variables.env

```diff
@@ -10,28 +10,28 @@ MEDIAWIKI_VERSION=1.40.1
 RELEASE_MAJOR_VERSION=1.40
 RELEASE_VERSION=1.40.1
 
-# WDQS_VERSION 0.3.135 is latest release on 2023-10-11
+# WDQS_VERSION 0.3.136 is latest release on 2023-11-09
 # as per https://github.com/wikimedia/wikidata-query-rdf/tags
 # and https://archiva.wikimedia.org/repository/releases/org/wikidata/query/rdf/service/
 WDQS_VERSION=0.3.135
```

Comment:
just checking: we are staying on the 0.3.135 even if there is a newer one, right? why?

Reply:
This was the version I did the product acceptance tests with. If we bumped the WDQS version now, I think we would need to do the product acceptance tests again. But you are right, I should have bumped the version before the product acceptance tests. This was an oversight on my side.